tox.ini

```diff
@@ -0,0 +1,6 @@
+[flake8]
+max-line-length = 119
```

Comment:
Do we need this? I don't see any especially long lines here?

Reply:
beats me I just copy pasted this from ocf/utils

Reply:
I'd try leaving it off and see if anything in pre-commit complains.

Reply:
Oh I remember. I think without this it was doing 80 cols or something.